core/org.eclipse.cdt.core/src/org/eclipse/cdt/core/build/CBuildConfiguration.java

```diff
@@ -518,15 +518,15 @@ public Process startBuildProcess(List<String> commands, IEnvironmentVariable[] e
 
 	/**
 	 * @return The exit code of the build process.
-	 * @since 7.5
+	 * @since 8.0
 	 */
 	protected int watchProcess(IConsole console, IProgressMonitor monitor) throws CoreException {
 		return launcher.waitAndRead(console.getInfoStream(), console.getErrorStream(), monitor);
 	}
 
 	/**
 	 * @return The exit code of the build process.
-	 * @since 7.5
+	 * @since 8.0
```

Comment:
Is it right to change these since tags?

Reply:
AFAICT this new API hasn't actually been released yet, that is why it bumped.

Reply:
Ah, fair enough :-)